Client/Frontend/Popover/PopoverController.swift

```diff
@@ -214,6 +265,15 @@ class PopoverController: UIViewController {
             containerView.arrowDirection = direction
         }
         
+        if contentController.extendEdgeIntoArrow {
+            switch containerView.arrowDirection {
+            case .up:
+                contentController.additionalSafeAreaInsets = UIEdgeInsets(top: PopoverUX.arrowSize.height, left: 0, bottom: 0, right: 0)
```

Comment:
Again, these are the same. Am I missing something here?

Reply:
this one can't be shortened easily, because two insets are changed depending on state, top and bottom

Reply:
Oh, I see, they are adjusting different things 👍